### ISSUE TITLE
TypeError: Cannot convert undefined or null to object when rendering without DOM node

### ISSUE THREAD

**Issue by dgcoffman** [sanniassin/react-input-mask#77]
When running Jest unit test that renders an InputElement:

```
 TypeError: Cannot convert undefined or null to object
        at Function.getPrototypeOf (<anonymous>)

      at Object.componentDidMount (node_modules/react-input-mask/build/InputElement.js:808:74)
```

`input` is null, causing `Object.getPrototypeOf(input)` to throw.
```
if (Object.getOwnPropertyDescriptor && Object.getPrototypeOf && Object.defineProperty) {
    var input = this.getInputDOMNode();
    var valueDescriptor = Object.getOwnPropertyDescriptor(Object.getPrototypeOf(input), 'value');
    this.canUseAccessors = !!(valueDescriptor && valueDescriptor.get && valueDescriptor.set);
}
```

This is blocking upgrading from 0.7.3 (which does not have this issue) to 0.7.9.

**Comment by dgcoffman** [comment]
See also: https://github.com/sanniassin/react-input-mask/issues/66

**Comment by sanniassin** [comment]
@dgcoffman Can you make a minimal project which will crash on ```npm run test```?

**Comment by pbarbiero** [comment]
Ran into this today as well. Worked around it for now by mocking the whole component.

**Comment by sanniassin** [comment]
@dgcoffman @pbarbiero Check this branch please https://github.com/sanniassin/react-input-mask/tree/jest-fix

**Comment by pbarbiero** [comment]
@sanniassin confirmed that works!

**Comment by sanniassin** [comment]
@pbarbiero Fixed in 0.8.2, thanks.
